lib/github_status_reporter.rb

```diff
@@ -22,11 +23,16 @@ class UnknonwnRepoCommit < StandardError; end
         'zooniverse/zoo-event-stats' => 'https://stats.zooniverse.org/'
       }.freeze
 
+      # Repos that do not use heads/master as their primary ref
+      PRIMARY_REF_BY_REPO = {
+      }.freeze
+
       JSON_COMMIT_ID_KEYS = %w[revision commit_id].freeze
       DEPLOYED_BRANCH_REPOS = {
         'zooniverse/talk-api' => 'production',
         'zooniverse/zoo-stats-api-graphql' => 'master',
-        'zooniverse/redsea' => 'main'
+        'zooniverse/redsea' => 'main',
```

Comment:
we don't use this repo anymore - it's archived
```suggestion
```

Reply:
✔️